docs/docs/contributor-swag.md

```diff
@@ -12,6 +12,8 @@ To show our appreciation, _**everyone who contributes to Gatsby is eligible to r
 
 If you contribute to the Gatsby org on GitHub, you can claim your swag by [logging into the Gatsby Maintainer Dashboard](https://store.gatsbyjs.org/login) and requesting a discount code.
 
+If you contribute five or more times, you can claim your _**Level 2**_ swag.
```

Comment:
Could you please add the sentence in the same paragraph, not as a new paragraph?
And I also like this sentence more:

`With five contributions or more you're eligible for Level 2 swag.`

Less repitition.